### ISSUE TITLE
Window size is restricted when moved to a larger monitor

### ISSUE THREAD

**Issue by mjfreelancing** [ForkIssues/TrackerWin#1286]
When Fork opens on my laptop and goes full screen, everything is good. When I then move the window to my external monitor the Fork window size is restricted. In the screen shot, the black edges show where the window is expanding but the content remains at a fixed size based on my laptop screen.

<img width="1021" alt="Screenshot 2021-08-24 124604" src="https://user-images.githubusercontent.com/17043225/130547706-0c701c7f-8f21-4318-b6f6-ac5d59e7bd93.png">

**Comment by DanPristupov** [comment]
I think this is a duplicate of https://github.com/fork-dev/TrackerWin/issues/1196

**Comment by mjfreelancing** [comment]
Yes. Sorry, I did browse the list but missed that one.On 24 Aug 2021 16:43, Dan ***@***.***> wrote:
I think this is a duplicate of #1196

**Comment by Remco-Schoeman** [comment]
Related issues: #942, #1286, #1196, #2458, #2736, #2484
